geomesa-tools/src/test/scala/org/locationtech/geomesa/tools/export/ExportCommandTest.scala

```diff
@@ -165,6 +165,25 @@ class ExportCommandTest extends Specification {
         readFeatures(format, file) mustEqual features.take(1)
       }
     }
+    "suppress empty output files" in {
```

Comment:
This test is great.  Since the first case does return empty files would it be worth changing the title to say something about testing both settings of the option?

Reply:
```suggestion
    "suppress or allow empty output files" in {
```